src/ImageGenerators/PvGenerator.h

```diff
@@ -0,0 +1,46 @@
+/* This file is part of the CARTA Image Viewer: https://github.com/CARTAvis/carta-backend
+   Copyright 2018, 2019, 2020, 2021 Academia Sinica Institute of Astronomy and Astrophysics (ASIAA),
+   Associated Universities, Inc. (AUI) and the Inter-University Institute for Data Intensive Astronomy (IDIA)
+   SPDX-License-Identifier: GPL-3.0-or-later
+*/
+
+#ifndef CARTA_BACKEND_IMAGEGENERATORS_PVGENERATOR_H_
+#define CARTA_BACKEND_IMAGEGENERATORS_PVGENERATOR_H_
+
+#include <casacore/images/Images/TempImage.h>
+#include <casacore/lattices/LRegions/LCRegion.h>
+
+#include <carta-protobuf/pv_request.pb.h>
+
+#include "../ImageData/FileLoader.h"
+#include "ImageGenerator.h"
+
+namespace carta {
+
+class PvGenerator {
+public:
+    PvGenerator(int file_id, const std::string& filename);
+    ~PvGenerator(){};
```

Comment:
Or just remove?  I think this is a refactoring relic.

I also think I don't need to have a map of pv generators in RegionHandler, since the cancellation is done in RegionHandler.  The pv generator is now a local variable after the profiles (pv data) are complete.